extensions/Worker.Extensions.CosmosDB/src/CosmosDBTriggerAttribute.cs

```diff
@@ -1,7 +1,7 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-﻿using System;
+using System;
```

Comment:
Is this a newline CRLF -> LF or something like that?

Reply:
I think so! It was showing up as a character that looked like this`<?>`